src/Og.php

```diff
@@ -59,4 +190,79 @@ public static function groupManager() {
     return \Drupal::service('og.group.manager');
   }
 
+  /**
+   * Invalidate cache.
+   *
+   * @param $gids
+   *   Array with group IDs that their cache should be invalidated.
+   */
+  public static function invalidateCache($gids = array()) {
+    // @todo We should not be using drupal_static() review and remove.
+    // Reset static cache.
+    $caches = array(
+      'og_user_access',
+      'og_user_access_alter',
+      'og_role_permissions',
+      'og_get_user_roles',
+      'og_get_permissions',
+      'og_get_group_audience_fields',
+      'og_get_entity_groups',
+      'og_get_membership',
+      'og_get_field_og_membership_properties',
+      'og_get_user_roles',
+    );
+
+    foreach ($caches as $cache) {
+      drupal_static_reset($cache);
+    }
+
+    // @todo Consider using a reset() method.
+    static::$entityGroupCache = [];
+
+    // Invalidate the entity property cache.
+    \Drupal::entityManager()->clearCachedDefinitions();
+    \Drupal::entityManager()->clearCachedFieldDefinitions();
+
+    // Let other OG modules know we invalidate cache.
+    \Drupal::moduleHandler()->invokeAll('og_invalidate_cache', $gids);
+  }
+
+  /**
+   * Gets the storage manage for the OG membership entity.
+   *
+   * @return \Drupal\Core\Entity\EntityStorageInterface
+   */
+  public static function membershipStorage() {
+    return \Drupal::entityManager()->getStorage('og_membership');
+  }
+
+  /**
+   * Gets the default constructor parameters for OG membership.
+   */
+  public static function membershipDefault() {
+    return ['type' => 'og_membership_type_default'];
+  }
+
+  /**
+   * Get all the modules fields that can be assigned to fieldable entities.
```

Comment:
=> `Get all the group audience fields ..`

It isn't the best name, but I never found a better one + not sure if we need to change this term. Option to discussion of course.